transmart-api-server/src/main/groovy/org/transmartproject/api/server/server/TransmartKeycloakAuthenticationProcessingFilter.groovy

```diff
@@ -0,0 +1,34 @@
+package org.transmartproject.api.server.server
+
+import groovy.util.logging.Slf4j
+import org.keycloak.adapters.springsecurity.filter.KeycloakAuthenticationProcessingFilter
+import org.springframework.security.authentication.AuthenticationManager
+import org.springframework.security.core.Authentication
+import org.springframework.security.core.AuthenticationException
+
+import javax.servlet.ServletException
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+/**
+ * This filter denies access to users without any roles assigned to them.
+ */
+@Slf4j
+class TransmartKeycloakAuthenticationProcessingFilter extends KeycloakAuthenticationProcessingFilter {
+
+    TransmartKeycloakAuthenticationProcessingFilter(AuthenticationManager authenticationManager) {
+        super(authenticationManager)
+        log.info("Enable filter that denies access to users without any roles assigned to them.")
```

Comment:
other token-verification logs (e.g. from the keycloak-client plugin) are on the error level, maybe useful to have this one as error too

Reply:
But error level should only be used for errors, right? these are not errors.